src/main/java/com/elastic/support/diagnostics/commands/VersionAndClusterNameCheckCmd.java

```diff
@@ -28,9 +29,9 @@ public boolean execute(DiagnosticContext context) {
          context.setClusterName(clusterName);
          String versionNumber = root.path("version").path("number").asText();
          context.setVersion(versionNumber);
-
       } catch (Exception e) {
-         logger.info("Error retrieving Elasticsearch version  - unable to continue..  Please make sure the proper connection parameters were specified", e.getMessage());
+         logger.info(e.getMessage());
+         logger.info("Error retrieving Elasticsearch version  - unable to continue..  Please make sure the proper connection parameters were specified.");
```

Comment:
Can you not do

```java
logger.info("message", e);
```

Or perhaps,

```java
logger.info(e, "message");
```

One variant should be valid and print the stacktrace with the exception messages.

Reply:
Actually no. In this case there are two separate logging appenders - one for the user that goes to the console and one that's being written to the diagnostic archive and sent up. The diag appearer is using a custom level to separate output that only comes back to us. In the past, showing customers that actual stack traces has ranged from having no impact to actually creating more issues. So I don't show them any of it, and at this stage create an archive with a manifest, any other artifacts and the log with the actual stack traces. 

There's more granular info if they actually get to running something. This is to address the case where they've  failed because they tried to use http instead of https. So in this case showing a user who's not aware they they are running secure sockets or need to specify a login and  password is unlikely to help much. But it does provide a bit more info in terms of the actual error for the support engineer.

Reply:
Ah, then in that case, then why print two separate messages?

Reply:
Because that's the user output and I wanted it on two separate lines. It looks better to me than appending the system line separator. Just my pref.

Reply:
Hard to argue with preferences.